src/project/internal/projectactionscontroller.cpp

```diff
@@ -1577,6 +1581,29 @@ void ProjectActionsController::warnScoreCouldnotBeSaved(const std::string& error
     interactive()->warning(muse::trc("project/save", "Your score could not be saved"), errorText);
 }
 
+void ProjectActionsController::warnScoreCorruptAfterSave(const Ret& ret)
+{
+    std::string errMessage = ret.toString();
+
+    std::any corruptedAfterMove = ret.data("corruptedAfterMove");
+    std::string originalFileCorruptedMessage = corruptedAfterMove.has_value() && std::any_cast<bool>(corruptedAfterMove)
+                                               ? "Unfortunately your original file has become corrupt.\n\n"
```

Comment:
Shouldn't this string be translatable too?

Reply:
Sure, thanks.

Reply:
What is the difference between trc, qtrc, mtrc?

Reply:
They return different types of strings:

* trc - std::string

* qtrc - QString

* mtrc - muse::String

The choice of the function depends on the situation. But in general, it is better to use the one that results in fewer string type conversions